packages/now-static-build/src/index.ts

```diff
@@ -59,26 +60,55 @@ function validateDistDir(
   const hash = isDev
     ? '#local-development'
     : '#configuring-the-build-output-directory';
-  const docsUrl = `https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build${hash}`;
-
-  const info = config.zeroConfig
-    ? '\nMore details: https://zeit.co/docs/v2/platform/frequently-asked-questions#missing-public-directory'
-    : `\nMake sure you configure the the correct distDir: ${docsUrl}`;
+  const link = config.zeroConfig
+    ? 'https://zeit.co/docs/v2/platform/frequently-asked-questions#missing-public-directory'
+    : `https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build${hash}`;
```

Comment:
This link doesn't work

https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build#local-development

https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build#configuring-the-build-output-directory

Perhaps @timothyis or @msweeneydev  know where the new docs are 🤔

Reply:
It's now all in the first link